plugins/org.eclipse.elk.alg.force/src/org/eclipse/elk/alg/force/ElkGraphImporter.java

```diff
@@ -228,8 +228,8 @@ public void applyLayout(final FGraph fgraph) {
         if (!kgraph.getProperty(CoreOptions.NODE_SIZE_FIXED_GRAPH_SIZE)) {
             ElkUtil.resizeNode(kgraph, width, height, false, true);
         }
-        kgraph.setProperty(CoreOptions.CHILD_AREA_WIDTH, width);
-        kgraph.setProperty(CoreOptions.CHILD_AREA_HEIGHT, height);
+        kgraph.setProperty(CoreOptions.CHILD_AREA_WIDTH, width - padding.getHorizontal());
```

Comment:
This does also seem to affect the not top-down case. Have you tested this?

Reply:
No, currently this property is only used by top-down layout. In general, the child area should not include the padding by definition.